nodejs/eks/cni/aws-k8s-cni.yaml

```diff
@@ -185,7 +185,7 @@ spec:
           name: run-dir
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: eniconfigs.crd.k8s.amazonaws.com
```

Comment:
The test failures suggest you need to define a schema here.. See an example here: https://github.com/argoproj/argo-helm/pull/605/files

Reply:
See https://github.com/pulumi/pulumi-eks/pull/683 which includes the latest config from upstream including missing schema from this.